packages/asset-service/src/generateAssetData/baseAssets.ts

```diff
@@ -1,8 +1,9 @@
+import * as caip from '@shapeshiftoss/caip'
 import { Asset } from '@shapeshiftoss/types'
 
 export const ethereum: Asset = {
-  assetId: 'eip155:1/slip44:60',
-  chainId: 'eip155:1',
+  assetId: caip.ethAssetId,
```

Comment:
stylistically prefer the plain import but obviously fine

Reply:
I was just saving some lines so it didn't split import across lines, could update, not a big deal for me either way